concept/thing/AttributeImpl.java

```diff
@@ -99,26 +99,22 @@
 
     public abstract static class Remote<VALUE> extends ThingImpl.Remote implements Attribute.Remote<VALUE> {
 
-        Remote(GraknClient.Transaction transaction, java.lang.String iid) {
+        Remote(Transaction transaction, java.lang.String iid) {
             super(transaction, iid);
         }
 
         @Override
         public final Stream<ThingImpl> getOwners() {
-            return thingStream(
-                    ConceptProto.Thing.Req.newBuilder().setAttributeGetOwnersReq(
-                            ConceptProto.Attribute.GetOwners.Req.getDefaultInstance()),
-                    res -> res.getAttributeGetOwnersRes().getThingsList()
-            );
+            return stream(Proto.Thing.Attribute.getOwners(getIID()))
```

Comment:
Something I notice is we now have to write `getIID` at least once in virtually every concept method - a duplication that was previously avoided by use of `thingStream` and lambda functions.

I think, with this architecture, that is unavoidable.